distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.web.cfg

```diff
@@ -70,4 +70,12 @@ org.ops4j.pax.web.ssl.protocols.excluded=SSLv3,TLSv1
 # Adjust the system's heap size in order to handle a greater number of threads
 org.ops4j.pax.web.server.maxThreads=300
 org.ops4j.pax.web.server.minThreads=10
-org.ops4j.pax.web.server.idleTimeout=60000
\ No newline at end of file
+org.ops4j.pax.web.server.idleTimeout=60000
+
+#######################################
+# Jetty Server Authenticator Settings
+#######################################
+
+# Set the authmethod/realmname so that our JettyAuthenticator is used
+org.ops4j.pax.web.default.authmethod=DDF
+org.ops4j.pax.web.default.realmname=DDF
```

Comment:
📝  Make sure this change is propagated properly to downstream projects, if need be. Mentioning because @brjeter had to fix this in a downstream change 😄.

Reply:
A fix shouldn't be needed for downstream projects as this allows us to use the Jetty Authenticator.
https://github.com/codice/ddf/blob/33c14f9e7d5876eee2c45f44429ae8674041cc5f/platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/SecurityAuthService.java#L25
 If we want to change the name to make more sense in all projects, we could make a ticket to get that done.

Reply:
@jhunzik, @bakejeyner was meaning that you'd have to include these pax-web properties in this particular config file if the config file is being overridden in downstream projects. You already have that downstream change ready to go, so we're good here 👍